.github/workflows/workflow-start.yml

```diff
@@ -0,0 +1,38 @@
+name: Workflow start
+# This workflow is only required for externally triggered jobs that need to manually
+# set the check status for a commit/PR
+
+on:
+  workflow_call:
+    inputs:
+      workflowName:
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  start:
+    runs-on: ubuntu-20.04
+    permissions:
+      statuses: write
+    steps:
+    - name: Start status check
+      uses: envoyproxy/toolshed/gh-actions/status@a6e1c951217efae1ac6b2bf32c5a9729976442b8
+      with:
+        authToken: ${{ secrets.GITHUB_TOKEN }}
+        context: ${{ inputs.workflowName }}
+        state: 'pending'
+        sha: ${{ inputs.sha }}
```

Comment:
to associate with the PR it needs to to use the actual head ref 

you can see the (current) one for this pr eg with:

```console
$ git ls-remote upstream | grep 27371/ | grep head
c159a092033b26b4b2a5adf2cc97b1b5798af43c	refs/pull/27371/head
```

this is what is being passed

Reply:
seems i didnt understand correctly and the question is more one of `workflow.inputs` i think the `workflowName` is only necessary due to it being different in the `workflow_call` - but otherwise it gets the same `inputs`